### DISCUSSION TITLE
Config for TimeRangePicker to be optimized picking Year||Month||Week||Day as one entity (no from - to)

### DISCUSSION THREAD

**Discussion by haemse** [grafana/grafana#45165] [Dashboards, Panels & Visualizations]
In many cases Users might provide Dashboards to simple Users to view common time rang entities like a whole certain Year or Month or Week ... you get it.

Would be a big improvement to offer a possibility to configure Time Picker to just pick one specific Day (that will then be the range)
or of course
Year, Month - as said.

This would be a much much easier time picker for a lot of users and usecases - no dealing with all the presets, or from - to ... 00:00:00 to 23:59:59:999 .... and such. 

From an End User perspective.

Just simply as an option to configure the time range picker.
In general, i think it makes sense to make users choose on what to provide in the time range picker - to adjust it to the needs of the provided dashboard, and keep it simple with just the options necessary.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
